src/field/pokemon.ts

```diff
@@ -4222,7 +4222,7 @@ export interface AttackMoveResult {
 }
 
 export class PokemonSummonData {
-  public battleStats: integer[] = [ 0, 0, 0, 0, 0, 0, 0 ];
+  public battleStats: integer[] = [ 0, 0, 0, 0, 0, 0, 0 ]; // [Atk, Def, SpAtk, SpDef, Spd, Acc]
```

Comment:
```suggestion
  public battleStats: integer[] = [ 0, 0, 0, 0, 0, 0, 0 ]; // [Atk, Def, SpAtk, SpDef, Spd, Acc, Eva]
```
Not 100% sure on it being accuracy and then evasion but regardless, it's missing.

Reply:
looking at `data/battle-stat.ts` seems to support your guess